sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -324,6 +324,15 @@ def detect_release
       log_error("Error detecting release", e, debug: debug)
     end
 
+    def csp_report_uri
+      if dsn && dsn.valid?
+        uri = dsn.csp_report_uri
+        uri += "&sentry_release=#{release}" if release && !release.empty?
+        uri += "&sentry_environment=#{environment}" if environment && !environment.empty?
```

Comment:
I'd expect we URL-encode `release` and `environment` to avoid creating malformed URI (and some form of URL-injection).

Reply:
Updated, thanks 👍